pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py

```diff
@@ -358,6 +358,48 @@ def test_ptp_default_acc_scale(self):
         # +++++++++++++++++++++++
         self.assertEquals(EXP_VEL_SCALE*EXP_VEL_SCALE, req.max_acceleration_scaling_factor)
 
+    def test_ptp_global_scale(self):
+        """ Test the default acceleration scaling factor for ptp commands.
+
+            Test sequence:
+                1. Create ptp command.
+                2. Call convert function.
+                3. Set the scaling factor for the robot
+                4. Call the convert function.
+
+            Test results:
+                1. -
+                2. Correct default acceleration scaling factor is set.
```

Comment:
This makes the test `test_ptp_default_acc_scale` redundant. Probably also the case for the other commands.

Reply:
Right but yet I wanted to seperate the two since this is a dedicated feature...

Reply:
Well then skip 2. here, because the default behaviour is already tested.